src/Illuminate/Database/Query/Builder.php

```diff
@@ -770,6 +770,10 @@ protected function addArrayOfWheres($column, $boolean, $method = 'where')
         return $this->whereNested(function ($query) use ($column, $method, $boolean) {
             foreach ($column as $key => $value) {
                 if (is_numeric($key) && is_array($value)) {
+                    $value = array_pad($value, 3, null);
```

Comment:
This is needed because you can skip the operator to use the default `=`. Therefore we need to pad the array to the size of three to get the `$boolean` in the right position.